lib/playlist_id_migration_directions.md

```diff
@@ -0,0 +1,9 @@
+## Translate playlists from [this file](https://docs.google.com/spreadsheets/d/1gHHsqbiZNxeYeWApM7GhU4L8wiZuIJP3EvCd_hXk8xk/edit?usp=sharing) these ids were pulled to create the playlist id array. Some of the numbers in the google sheet aren't the most recent version of that playlist, so the spreadsheet and array don't exactly match. This file was last updated 11/14/17 - any playlists added to this file after that date are not included in this array. 
```

Comment:
This is a little silly, but you might replace this with a title instead of a whole description in a header tag:

```
## Translate playlists notes

These are instructions on how to translate playlists into casebooks from (google spreadsheet). Note that these ids were pulled...

1. First step...
2. Second step...
3. etc.
```